stko/optimizers/collapser.py

```diff
@@ -149,70 +152,70 @@ def _get_new_position_matrix(self, mol, step, vectors, scales):
 
         return new_position_matrix
 
-    def _get_BB_vectors(self, mol, BB_atom_ids):
+    def _get_bb_vectors(self, mol, bb_atom_ids):
         """
-        Get the BB to COM vectors.
+        Get the bb to COM vectors.
 
         Parameters
         ----------
         mol : :class:`.Molecule`
             The molecule to be optimized.
 
-        BB_atom_ids : :class:`dict`
+        bb_atom_ids : :class:`dict`
             Atom ids (values) in each distinct building block in the
             molecule. Keys are building block ids.
 
         Returns
         -------
-        BB_cent_vectors : :class:`dict`
+        bb_cent_vectors : :class:`dict`
             Vector from building block (Key is building block id) to
             molecules centroid.
 
-        BB_cent_scales : :class:`dict`
+        bb_cent_scales : :class:`dict`
             Relative size of vector between building blocks and
             centroid.
 
         """
 
         cent = mol.get_centroid()
 
-        # Get BB COM vector to molecule COM.
-        BB_cent_vectors = {
-            i: mol.get_centroid(atom_ids=BB_atom_ids[i])-cent
-            for i in BB_atom_ids
+        # Get bb COM vector to molecule COM.
+        bb_cent_vectors = {
+            i: mol.get_centroid(atom_ids=bb_atom_ids[i])-cent
+            for i in bb_atom_ids
         }
 
         # Scale the step size based on the different distances of
-        # BBs from the COM. Impacts anisotropic topologies.
+        # bbs from the COM. Impacts anisotropic topologies.
         if self._scale_steps:
             max_distance = max(
-                np.linalg.norm(BB_cent_vectors[i])
-                for i in BB_cent_vectors
+                np.linalg.norm(bb_cent_vectors[i])
+                for i in bb_cent_vectors
             )
-            BB_cent_scales = {
-                i: np.linalg.norm(BB_cent_vectors[i])/max_distance
-                for i in BB_cent_vectors
+            bb_cent_scales = {
+                i: np.linalg.norm(bb_cent_vectors[i])/max_distance
```

Comment:
Code repetition here. You should store the result np.linalg.norm as a variable as use it later on

Reply:
Done.